dist/rpm/os-autoinst-openvswitch-test.spec

```diff
@@ -19,7 +19,7 @@ touch %{_sourcedir}/%{short_name}
 %build
 # call one of the components but not openqa itself which would need a valid
 # configuration
-/usr/lib/os-autoinst/os-autoinst-openvswitch --help
+/usr/lib/os-autoinst/script/os-autoinst-openvswitch --help
```

Comment:
What's the context here? Was there a related change that broke tests? If not, how did you find out this was broken?

Reply:
I guess this might be related to #1978 - no CI failures there 🤔

Reply:
Yes,
* #1978

Reply:
Apparently I can't view OBS test results for that, or this PR.